app.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "VoteIT",
+  "description": "A voting system",
+  "repository": "https://github.com/cthit/VoteIT",
+  "logo": "http://imgur.com/3Xwg8XF.png",
```

Comment:
Perhaps shouldn't refer to logo on imgur? Better to refer to one on GitHub or Chalmers.it

Reply:
Where is this logo used?